test/e2e/framework/network_pods.go

```diff
@@ -31,7 +31,7 @@ func (f *Framework) CreateTCPCheckListenerPod(cluster int, sendString string) *v
 				{
 					Name:    "tcp-check-listener",
 					Image:   "busybox",
-					Command: []string{"sh", "-c", "echo listener says $SEND_STRING | nc -l -v -p $LISTEN_PORT -s 0.0.0.0 >/dev/termination-log 2>&1"},
+					Command: []string{"sh", "-c", "for i in $(seq 50); do echo listener says $SEND_STRING; done | nc -l -v -p $LISTEN_PORT -s 0.0.0.0 >/dev/termination-log 2>&1"},
```

Comment:
Here I'm sending the string more times, because sometimes it will fail in travis CI otherwise (likely to be due to resource pressure).

In the failure cases, we could see the connection happening from one pod to the other in the logs, but no data was received in one of the directions.

Reply:
Shouldn’t this be mentioned in the code itself?

Reply:
Done, thanks @skitt